Makefile

```diff
@@ -33,7 +33,7 @@ clean:
 	go clean .
 	go clean ./cmd/lit-af
 	rm -rf build/_workspace/
-	rm -f cmd/lit-af/lit-af
+	rm -f lit cmd/lit-af/lit-af lit-af
```

Comment:
I think the last `lit-af` can be removed since that doesn't get built in `/`?

Reply:
It does when you build with `make`.